CompilerSource/parser/collect_variables.cpp

```diff
@@ -234,6 +235,8 @@ class DeclGatheringVisitor : public AST::Visitor {
         prefix = ftype.substr(0, pos);
         suffix = ftype.substr(pos + name.length());
       }
+      prefix = std::regex_replace(prefix, std::regex("^ +| +$|( ) +"), "$1");
+      suffix = std::regex_replace(suffix, std::regex("^ +| +$|( ) +"), "$1");
```

Comment:
Not doing that will prevent generating `case obj_0: return ((OBJ_obj_0*)inst)->val;` (for `local var val = 12;`) for some reason.